### PR TITLE
Fix typos in messages pod rm

### PR DESCRIPTION
**Pull request by sujil02**
Fix typos in messages

Signed-off-by: Sujil02 <sushah@redhat.com>

### CODE REVIEW

File: pkg/domain/infra/abi/pods.go [hunk 1]
```diff
@@ -226,15 +226,14 @@ func (ic *ContainerEngine) PodRm(ctx context.Context, namesOrIds []string, optio
 		reports []*entities.PodRmReport
 	)
 	pods, err := getPodsByContext(options.All, options.Latest, namesOrIds, ic.Libpod)
-	if err != nil {
+	if err != nil && !(options.Ignore && errors.Cause(err) == define.ErrNoSuchPod) {
```
Comment (vrothberg) [inline pkg/domain/infra/abi/pods.go:229]: I believe we need to move ignore into `getPodsByContext`. If we specify three pods and the first is missing, we still want the other two. Does `podman pod rm bogus existingA existingB` work correctly and removes the last two?

Comment (vrothberg) [reply pkg/domain/infra/abi/pods.go:229]: I had a look at the code and it's fine. `getPodsByContext` runs over all specified ctrs/pods.

**Comment by vrothberg** [comment]
May already be fixed in master now.

**Comment by sujil02** [comment]
@giuseppe rm pod fixes got merged with #5997 should I close this one? however, it has some help message typos fixed.

**Comment by sujil02** [comment]
@rhatdan I accidentally pushed to this branch thought this was merged. no new changes are done. need lgtm again.

**Pull request merged**